test/MUXControlsReleaseTest/AppThatUsesMUXIndirectly/AppThatUsesMUXIndirectly.csproj

```diff
@@ -138,6 +138,9 @@
     <PackageReference Include="MUXCustomBuildTasks">
       <Version>1.0.67</Version>
     </PackageReference>
+    <PackageReference Include="RuntimeComponentThatUsesMUX">
```

Comment:
How does this project find RuntimeComponentThatUsesMUX.nupkg? I know that you updated the yml to build that sln first. But does the nupkg file get copied somewhere?

Reply:
The `AfterBuild` target in RuntimeComponentThatUsesMUX.vcxproj places it in localpackages, which is used as a package source in nuget.config.

Reply:
Ah. I see that now. Thanks.